content/en/blog/_posts/2022-03-31-Kubernetes-1-24-deprecations-and-removals

```diff
@@ -0,0 +1,85 @@
+
+For more information about why Kubernetes is moving away from dockershim, check out the aptly named: [Kubernetes is Moving on From Dockershim](/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/) and the [updated dockershim removal FAQ](/blog/2022/02/17/dockershim-faq/).
+
+Take a look at the [What you need to do to have a working cluster for 1.24](https://github.com/kubernetes/website/issues/30917) to learn what you need to do to go from a working 1.23 cluster to a working 1.24 cluster. 
+
+## What to do
+
+### `kubectl convert`
+
+As mentioned in [prior API removal posts](/blog/2021/07/14/upcoming-changes-in-kubernetes-1-22/#kubectl-convert), [`kubectl-convert`](/docs/tasks/tools/included/kubectl-convert-overview/),  a plugin for the `kubectl` Kubernetes command-line tool, can be helpful to address migrating off deprecated APIs. It does so by facilitating the conversion of manifests between different API versions, for example, from a deprecated to a non-deprecated API version. More general information about the API migration process can be found in the [Deprecated API Migration Guide](/docs/reference/using-api/deprecation-guide/).
+    
+## The Kubernetes API removal process
+
+Kubernetes contains a large number of components that evolve over time. In some cases, this evolution results in APIs, flags, or entire features, being removed. To prevent users from facing breaking changes, Kubernetes adopted a feature [deprecation policy](/docs/reference/using-api/deprecation-policy/). This policy ensures that stable APIs may only be deprecated when a newer stable version of that same API is available and that APIs have a minimum lifetime depending on their stability level: 
+
+* Generally available (GA) or stable API versions may be marked as deprecated but must not be removed within a major version of Kubernetes. 
+* Beta or pre-release API versions must be supported for 3 releases after deprecation. 
+* Alpha or experimental API versions may be removed in any release without prior deprecation notice. 
+
+Kubernetes will continue to remove APIs by following the deprecation policy regardless of whether an API is removed due to a beta feature graduating to stable or because that API was not proven to be successful and will also make sure migration options are documented.
```

Comment:
You can skip the “The Kubernetes API removal process” heading. The key thing being removed in v1.24 is Docker Engine support, which is not an API.

Reply:
Are you suggesting I cut this section entirely?